superset/security.py

```diff
@@ -127,12 +127,11 @@ def all_datasource_access(self):
         return self.can_access(
             'all_datasource_access', 'all_datasource_access')
 
+    def all_database_access(self):
+        return self.can_access('all_database_access', 'all_database_access')
```

Comment:
In the original PR, this was sending a `user` keyword argument. However, `can_access()` does not currently accept a keyword arg so I removed it